docs/en/sql-reference/functions/time-series-functions.md

```diff
@@ -8,6 +8,57 @@ sidebar_label: Time Series
 
 Below functions are used for time series analysis.
 
+## seriesOutliersTukey
+
+Detects a possible anomaly in series using [Tukey Fences](https://en.wikipedia.org/wiki/Outlier#Tukey%27s_fences).
```

Comment:
`Detects outliers in series data using [...]`

Kind of related to this PR: Could you please replace occurrences of `time series` on this markdown page by `series data` (the input is strictly speaking not related to time series)